Sources/PackageGraph/Pubgrub/PubgrubDependencyResolver.swift

```diff
@@ -102,30 +102,30 @@ public struct PubgrubDependencyResolver {
     /// The container provider used to load package containers.
     private let provider: ContainerProvider
 
-    /// Skip updating containers while fetching them.
-    private let skipUpdate: Bool
-
     /// Reference to the package container provider.
     private let packageContainerProvider: PackageContainerProvider
 
     /// Should resolver prefetch the containers.
-    private let isPrefetchingEnabled: Bool
+    private let prefetchingEnabled: Bool
+
+    /// Skip updating containers while fetching them.
```

Comment:
Since the sense of the boolean has flipped, the comment should probably change too?
```suggestion
    /// Allow updating containers while fetching them.
```